projects/swimlane/ngx-ui/src/lib/services/injection/injection.service.ts

```diff
@@ -143,7 +143,7 @@ export class InjectionService {
     const appendLocation =
       bindings.inputs && bindings.inputs.isRoot === false ? location : this.getComponentRootNode(location);
 
-    const portalHost = new DomPortalHost(
+    const portalHost = new DomPortalOutlet(
```

Comment:
it really was as simple as changing this to outlet huh

Reply:
Sure was, lucky my personal project had a working example to go off.